app/src/main/java/com/example/plantsapp/presentation/ui/utils/ImageLoadingExt.kt

```diff
@@ -31,12 +31,7 @@ fun Uri?.getBitmapWithPlaceholder(
         .disableCache()
         .load(this ?: placeholder)
         .submit()
-        .run {
-            val bitmap = this.get()
-            Glide.with(context).clear(this)
```

Comment:
removed this cause I kept getting his error
![Screen Shot 2021-11-29 at 6 43 10 PM](https://user-images.githubusercontent.com/43578259/143898851-369baa35-afc3-4360-95e8-fa625417353b.png)
![Screen Shot 2021-11-29 at 6 48 37 PM](https://user-images.githubusercontent.com/43578259/143899097-08fceaa4-390f-4177-91b6-24f12ee05f44.png)
)